src/app/store.ts

```diff
@@ -1,6 +1,10 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
 
-const rootReducer = combineSlices();
+import { todosSlice } from '../features/todos';
+import { currentTodoSlice } from '../features/currentTodo';
+import { filterSlice } from '../features/filter';
+
+const rootReducer = combineSlices(todosSlice, currentTodoSlice, filterSlice);
```

Comment:
Replace `combineSlices` with `combineReducers` to correctly combine the slices into a root reducer.